src/components/common/Navbar/Navbar.js

```diff
@@ -19,8 +19,19 @@ const NAV_ITEMS = ['About', 'Partners', 'Team', 'FAQ'];
 class Navbar extends Component {
   state = {
     mobileMenuOpen: false,
+    opacityIncrement: 0,
   };
 
+  componentDidMount() {
```

Comment:
Can you add some comments here and in the render function to explain what this is doing. Just for future reference.

Reply:
👍